oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -366,10 +489,23 @@ public Builder setQuotaProjectId(String quotaProjectId) {
       return this;
     }
 
+    public Builder setUniverseDomain(String universeDomain) {
+      this.universeDomain = universeDomain;
+      return this;
+    }
+
     public String getQuotaProjectId() {
       return this.quotaProjectId;
     }
 
+    public String getUniverseDomain() {
+      if (this.universeDomain == null || this.universeDomain.trim().isEmpty()) {
```

Comment:
nit: I feel like it's more correct to do the defaulting when calling "build"; this way I can tell if a builder has a universe domain set or not.

Reply:
Agree, thanks!